src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -96,6 +96,15 @@ CorProfiler::Initialize(IUnknown* cor_profiler_info_unknown) {
     return E_FAIL;
   }
 
+  // get ICorProfilerInfo6 for net452+
+  ICorProfilerInfo6* info6;
```

Comment:
This comment seems wrong. Is it net46+?

Reply:
Oh, right

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1542/commits/338ec8db17cd8aa5198215743e2ce066ead20672